gradio/__init__.py

```diff
@@ -96,7 +96,7 @@
     TextArea,
 )
 from gradio.themes import Base as Theme
-from gradio.utils import NO_RELOAD, get_package_version, set_static_paths
+from gradio.utils import NO_RELOAD, FileSize, get_package_version, set_static_paths
```

Comment:
Why do we export FileSize?

Reply:
So users can use ` 5 * FileSize.MB`

Reply:
got it. that reminds me, we should add documentation for `max_file_size` in one of the guides. Perhaps https://www.gradio.app/guides/sharing-your-app#security-and-file-access

Reply:
Great idea - just pushed up!